packages/rum-core/src/domain/view/viewCollection.ts

```diff
@@ -126,7 +126,9 @@ function processViewUpdate(
       toServerDuration as (duration: Duration) => ServerDuration
     )
   }
+
   return {
+    customerContext: view.context,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You could simplify the logic a bit:
- remove the customerContext:
```suggestion
```
- Always merge `viewContext.context` in  assembly.ts: 
```ts 
serverRumEvent.context = combine(
     commonContext.context,
     viewContext.context,
     customerContext,
)
```